Configurations/shared-info.pl

```diff
@@ -27,6 +27,7 @@ sub detect_gnu_cc {
     'gnu-shared' => {
         shared_ldflag         => '-shared -Wl,-Bsymbolic',
         shared_sonameflag     => '-Wl,-soname=',
+        rpath_flag            => '-Wl,-rpath=',
```

Comment:
Oh!  I was more thinking that this would be moved to those shared targets that were referred to from 10-main.conf, such as `linux-shared`...  but if you think this works, then fine :-)